web/cobrands/peterborough/assets.js

```diff
@@ -225,9 +225,9 @@ fixmystreet.assets.add(flytipping_defaults, {
                     return;
                 }
             }
-            $("#js-roads-responsibility").removeClass("hidden");
-            $("#js-roads-responsibility .js-responsibility-message").addClass("hidden");
+            $("#js-roads-responsibility").addClass("hidden");
```

Comment:
This line, what is it for now? If this is using its own message to display, why is this concerned with js-roads-responsibility at all? (Or if for some reason I've missed it is needed, why was it removed from the found function above.)